cli/CMakeLists.txt

```diff
@@ -44,6 +44,20 @@ configure_file (
   "${CMAKE_CURRENT_BINARY_DIR}/CliConfig.h"
 )
 
+# Compile the protos for Single Node Server mode.
+if(ENABLE_SINGLE_NODE_SERVER)
+  # We will need some of the TMBs libraries
+  set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/third_party/src/tmb/cmake")
+
+  find_package(Grpc++ REQUIRED)
+  include_directories(${GRPCPLUSPLUS_INCLUDE_DIRS})
```

Comment:
Seems to me like leaving it in `/cli/cmakelists.txt` should be fine because the module is being used by sources in `/cli/` but not necessarily by sources in `/`.